definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -5355,13 +5341,6 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "12/08/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solAosReceivedNoAdConStarted",
```

Comment:
Just need to keep this original object in "AuthorisationCaseEvent-prod.json".

Reply:
I completely agree with this. And as far as i can tell, this is the correct approach. Im waiting for Jeremy to get back to me but im going to impliment as you've mentioned

Reply:
Update : cec1c1a5d43c88a86a4af17a0a4f1aed1b6bae78